src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -53,7 +54,7 @@ public class Win32Program : IProgram
 
         public bool Enabled { get; set; }
 
-        public bool HasArguments { get; set; }
+        public bool HasArguments => !string.IsNullOrEmpty(Arguments);
```

Comment:
This parameter can easily be forgotten, this will calculate the HasArgument at runtime